input/pagecontent/reference.md

```diff
@@ -2830,6 +2842,7 @@ Following [standard profiling rules established in FHIR](https://www.hl7.org/fhi
 |  OID   | Object Identifier
 | $SCT | Common FSH alias for SNOMED Clinical Terms
 |  SU  | Flag denoting "include in summary"
+|  SD  | StructureDefinition
```

Comment:
I noticed you use `SD` in a few places (so I also felt free to use it).  But it probably should be listed in the acronyms.